pkg/apis/wmc/v1alpha1/windowsmachineconfig_types.go

```diff
@@ -11,6 +11,8 @@ import (
 type WindowsMachineConfigSpec struct {
 	// Replicas represent how many Windows nodes to be added to the
 	// OpenShift cluster
+	// +kubebuilder:validation:Required
```

Comment:
This didnt cause a change in the CRD. is that expected?

Reply:
Good catch. Removed it. Thanks

Reply:
This should've  Minimum:0 as well